modules/oe_whitelabel_paragraphs/oe_whitelabel_paragraphs.module

```diff
@@ -234,11 +236,20 @@ function oe_whitelabel_preprocess_paragraph__oe_list_item_block(array &$variable
   $variables['columns'] = ['two_columns' => '2', 'three_columns' => '3'][$layout_name] ?? '1';
 
   $variables['items'] = [];
-  foreach ($variables['paragraph']->get('field_oe_paragraphs') as $card_paragraph_item) {
+  foreach ($variables['paragraph']->get('field_oe_paragraphs') as $sub_paragraph) {
     /** @var \Drupal\paragraphs\ParagraphInterface $card_paragraph */
-    $card_paragraph = $card_paragraph_item->entity;
-    $card_image_item = $card_paragraph->get('field_oe_image')->first();
-    $card_image = $card_image_item ? ImageValueObject::fromImageItem($card_image_item) : NULL;
+    // Get the paragraph translation.
+    $card_paragraph = \Drupal::service('entity.repository')
```

Comment:
/**
  * The entity repository.
  *
  * @var \Drupal\Core\Entity\EntityRepositoryInterface
   */
  protected $entityRepository;

and use this $entityRepository variable.

Never use "\Drupal::service(" inside a class due to jection dependencies. For sample take a look here https://www.drupal.org/project/drupal/issues/3344389